tools/ports/benchmark/portfile.cmake

```diff
@@ -0,0 +1,49 @@
+if(VCPKG_CMAKE_SYSTEM_NAME STREQUAL "WindowsStore")

```

Comment:
Do we plan to upstream the port to official vcpkg? That will make opentelemetry-cpp easier to be consumed.

Reply:
Yes, absolutely.

Reply:
This one was a "hack" to make sure we fix the compat issue in Google Benhmark, i.e. allow the benchmark build with vs2019 to be used by vs2017 compile. I'll see what we can do, if I can contribute it as an issue / recommending the patch we applied here..